web/packages/teleport/src/Roles/RoleEditor/StandardEditor/StandardEditor.tsx

```diff
@@ -112,12 +106,7 @@ export const StandardEditor = ({
     }
     validator.reset();
     setCurrentTab(nextTabIndex);
-    setDisabledTabs(prevEnabledTabs =>
-      produce(prevEnabledTabs, et => {
-        et[nextTabIndex] = false;
-      })
-    );
-  }, [currentTab, validate, validator, setCurrentTab, setDisabledTabs]);
+  }, [currentTab, validate, validator, setCurrentTab]);
```

Comment:
`setCurrentTab` is a dependency, but is not wrapped in `useCallback` which makes memoization here useless.

But instead of adding it to `setCurrentTab`, I'd remove it here. `validateAndGoToNextTab` is called only in a button, so we don't need to memoize it.

Reply:
Thanks for pointing it out. I'll actually add `useCallback` for `setCurrentTab`, as I remember having to do a lot of performance optimization in the standard editor, so every button counts.